test/libsolidity/syntaxTests/viewPureChecker/tload_view.sol

```diff
@@ -0,0 +1,10 @@
+contract C {
+    function f() external view returns (uint a) {
+        assembly {
+            a := tload(0)
```

Comment:
Need to add `tload` to 
```
syntaxTests/viewPureChecker/inline_assembly_instructions_allowed_view_cancun.sol
```
You can even delete this test afterwards, since it would be redundant.

Reply:
Done.